### PR TITLE
o/snapstate: remove unused slice that is only appended to

### PR DESCRIPTION
**Pull request by andrewphelpsj**
Removes an unused slice, unsure what it was originally used for.

### CODE REVIEW

**Comment by andrewphelpsj** [comment]
Actually, after reading this undo handler more carefully, does it even do anything?

**Review by MiguelPires — COMMENTED** [review]
I see:
```
	// TODO: telemetry about errors here
```
at the end of the handler so I'm guessing this was meant to collect information about the error to eventually do something with it? But it was added in 2017 so the plan might've changed. Maybe @pedronis remembers

File: overlord/snapstate/handlers.go [hunk 1]
```diff
@@ -629,21 +629,17 @@ func (m *SnapManager) undoPrepareSnap(t *state.Task, _ *tomb.Tomb) error {
 		return nil
 	}
 
-	var logMsg []string
 	var snapSetup string
 	dupSig := []string{"snap-install:"}
```
Comment (bboozzoo) [inline overlord/snapstate/handlers.go:633]: dupSig appears to be equally useless?

**Review by pedronis — CHANGES_REQUESTED** [review]
in theory we can empty the whole definition and just leave the TODO about telemetry for errors, a different approach would be to replace the code with a short text that list what we were/would want to capture

at some past point this code was sending the computed info to the errortracker but we removed this because that's wasn't working/scaling for us (afair)

**Comment by Meulengracht** [comment]
@andrewphelpsj would you mind updating the status of this PR?

**Comment by andrewphelpsj** [comment]
@Meulengracht I took @pedronis' suggestion and just removed the handler with a TODO about telemetry for now.

**Review by bboozzoo — APPROVED** [review]
Thank you

**Review by MiguelPires — APPROVED** [review]
lgtm

**Review by pedronis — APPROVED** [review]
+1

**Pull request merged**